nixos/tests/age-plugin-tpm-decrypt.nix

```diff
@@ -0,0 +1,33 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+  {
+    name = "age-plugin-tpm-decrypt";
+    meta = with pkgs.lib.maintainers; {
+      maintainers = [ sgo ];
```

Comment:
@stigtsp how should I fill this out? I just put your name here for now. I'm not registered in the maintainers list yet, but happy to do so if you'd prefer.

Reply:
Awesome! I'd be honored